layers/meta-resin-genericx86/recipes-support/hostapp-update-hooks/files/99-flash-grub

```diff
@@ -10,20 +10,23 @@ getSize() {
   echo $(du -sb "$1" | awk '{print $1}')
 }
 
-MBR=512
-reservedMBR=66
-sourcePath="/resin-boot/grub/"
-device="/dev/$(findmnt --noheadings --canonicalize --output SOURCE /mnt/sysroot/active | xargs lsblk -no pkname)"
+# flash legacy grub only if we do not support UEFI
+if ! efibootmgr &>/dev/null ; then
```

Comment:
This introduces a runtime dependency on efibootmgr for this script which is not specified in Yocto metadata.